CMakeLists.txt

```diff
@@ -964,6 +976,7 @@ if (BUILD_TESTING)
   # Fixture requirements
   set_tests_properties (cleanup_immediately PROPERTIES FIXTURES_REQUIRED logcleanuptest)
   set_tests_properties (cleanup_with_prefix PROPERTIES FIXTURES_REQUIRED logcleanuptest)
```

Comment:
It would be adequate now to rename the test and related files to `cleanup_with_absolute_prefix`.